vip-mail.php

```diff
@@ -8,6 +8,33 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+if ( ! class_exists( 'PHPMailer\PHPMailer\PHPMailer' ) ) {
+	require_once ABSPATH . WPINC . '/PHPMailer/PHPMailer.php';
+	require_once ABSPATH . WPINC . '/PHPMailer/SMTP.php';
+	require_once ABSPATH . WPINC . '/PHPMailer/Exception.php';
+}
+class VIP_PHPMailer extends PHPMailer\PHPMailer\PHPMailer {
+	/**
+	 * Check whether a file path is of a permitted type.
+	 *
+	 * Used to reject URLs and phar files from functions that access local file paths,
+	 * such as addAttachment. Allows VIP File System's `vip` protocol.
+	 *
+	 * @param string $path A relative or absolute path to a file
+	 *
+	 * @return bool
+	 */
+	protected static function isPermittedPath( $path ) {
+		if ( 0 === strpos( $path, 'vip://' ) ) {
```

Comment:
```suggestion
		if ( 0 === strpos( $path, 'vip://wp-content/uploads' ) ) {
```
Stream Wrapper only supports paths that are starting with `wp-content/uploads`, let's match that here.